packages/cli/src/services/linter/utils/getRuleset.ts

```diff
@@ -20,6 +24,10 @@ async function getDefaultRulesetFile(): Promise<Optional<string>> {
   return;
 }
 
+function isLegacyRuleset(filepath: string): boolean {
```

Comment:
Not a huge deal, but I hesitate to start calling these "legacy" rulesets - `isBasicRuleset`, `isStructuredRuleset`, `isNotJSRuleset`, etc etc?

Reply:
I like the name `isBasicRuleset`! <!-- but honestly these are a bit legacy :troll: -->